frontend/public/components/operator-hub/operator-hub-page.tsx

```diff
@@ -14,16 +14,14 @@ import { PackageManifestKind, OperatorGroupKind, SubscriptionKind } from '../ope
 import { installedFor, subscriptionFor } from '../operator-lifecycle-manager/operator-group';
 import { OperatorHubItem, OperatorHubCSVAnnotations } from './index';
 
-import * as operatorImg from '../../imgs/operator.svg';
-
 export const OperatorHubList: React.SFC<OperatorHubListProps> = (props) => {
   const {operatorGroup, subscription, loaded, loadError, namespace = ''} = props;
   const marketplaceItems = _.get(props.marketplacePackageManifest, 'data', [] as PackageManifestKind[]);
   const localItems = _.get(props.packageManifest, 'data', [] as PackageManifestKind[]);
   const items = marketplaceItems.concat(localItems).map(pkg => {
     const currentCSVDesc = _.get(pkg, 'status.channels[0].currentCSVDesc', {});
     const currentCSVAnnotations: OperatorHubCSVAnnotations = _.get(currentCSVDesc, 'annotations', {});
-    const iconObj = _.get(currentCSVDesc, 'icon[0]');
+    const {name, currentCSV} = pkg.status.channels[0];
```

Comment:
Will `status.channels` always be set? I see we use `_.get` above

Reply:
If we're going to destructure, we should add `currentCSVDesc` from the `_.get` call above

Reply:
Done.